server/assessment/service/annual/annualYearsExporter.js

```diff
@@ -3,65 +3,61 @@ const Promise = require('bluebird')
 const CSVOutputWithVariables = require('../csvOutputWithVariables')
 
 const CountryConfigExporter = require('../exporter/countryConfigExporter')
-//1a
+// 1a
 const ExtentOfForestExporter = require('../fraYears/section_1/extentOfForestExporter')
 
-//5
+// 5
 const DisturbancesExporter = require('./section_5/disturbancesExporter')
 const AreaAffectedByFireExporter = require('./section_5/areaAffectedByFireExporter')
 
 const YEARS_ANNUAL = R.range(2000, 2018)
 
-const fetchCountryData = async countryIso => await Promise.all([
-  CountryConfigExporter.fetchData(countryIso),
-  //1a,
-  ExtentOfForestExporter.fetchData(countryIso),
-  //5a, 5b
-  DisturbancesExporter.fetchData(countryIso),
-  AreaAffectedByFireExporter.fetchData(countryIso),
+const fetchCountryData = async (countryIso) =>
+  await Promise.all([
+    CountryConfigExporter.fetchData(countryIso),
+    // 1a,
+    ExtentOfForestExporter.fetchData(countryIso),
+    // 5a, 5b
+    DisturbancesExporter.fetchData(countryIso),
+    AreaAffectedByFireExporter.fetchData(countryIso),
+  ])
 
-])
-
-const getCountryData = async country => {
+const getCountryData = async (country) => {
   const [
     countryConfig,
-    //1a
+    // 1a
     extentOfForest,
-    //5a
-    disturbances, areaAffectedByFire
+    // 5a
+    disturbances,
+    areaAffectedByFire,
   ] = await fetchCountryData(country.countryIso)
 
   // iterate over years
   return YEARS_ANNUAL.map((year, yearIdx) => ({
     ...country,
     year,
 
-    //forestArea2020
-    forestArea2020: R.pipe(
-      R.prop('fra'),
-      R.find(R.propEq('year', 2020)),
-      R.propOr('', 'forestArea')
-    )(extentOfForest),
+    // forestArea2020
+    forestArea2020: R.pipe(R.prop('fra'), R.find(R.propEq('year', 2020)), R.propOr('', 'forestArea'))(extentOfForest),
 
-    //country config
+    // country config
     ...CountryConfigExporter.parseResultRow(countryConfig, yearIdx, year),
-    //5a, 5b
+    // 5a, 5b
     ...DisturbancesExporter.parseResultRow(disturbances, yearIdx, year),
     ...AreaAffectedByFireExporter.parseResultRow(areaAffectedByFire, yearIdx, year),
   }))
-
 }
 
-const getCsvOutput = () => {
+const getCsvOutput = (noVariableFolder) => {
```

Comment:
can we call this `includeVariableFolders` ? it's more meaningful

Reply:
yes, but I didnt want to touch the old code - I can change this

Reply:
It is used widely, leaving as is if ok

Reply:
but I don't understand what noVariableFolder means. can you explain ?

Reply:
It is a toggle variable for excluding the subfolders in the zip file

Reply:
I can always rename it to `includeVariableFolders` and set the default value to true